src/database/resolver.test.js

```diff
@@ -1,5 +1,27 @@
-describe('resolver test demo', () => {
-	it('demo test', () => {
-		expect(2 + 2).toBe(4)
+const { graphqlTestCall } = require('./graphqlTestCall')
+const { dbConnection } = require('../helper/connectionHelper')
```

Comment:
Since we don't want our CI servers hitting our database while running tests, we should put integration tests that touches external systems (e.g. database, external api) into files with names like `[name].integration.js` so that we can run it locally but exclude it from running in CI using `testMatch`

Reply:
yup we need to follow that convention.